swagger-v1/swagger.yaml

```diff
@@ -2364,19 +2374,26 @@ components:
         Note: The credentialType is required and must be set to 'Express'
     PaymentGatewayRaftCredentials:
       required:
+      - bankId
```

Comment:
bankId needs to be changed to stpBankId or else it won't show up as a required field aka it's missing the star next to it